docs/site/tutorials/connectors/Connecting-to-back-end.md

```diff
@@ -0,0 +1,16 @@
+---
+lang: en
+title: 'Connecting to Back-end Service Tutorials'
+keywords:
+  LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Tutorial, Connector, Datasource
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Connecting-to-back-end.html
+---
+
+<!-- TODO
```

Comment:
Please add a short note to explain readers that this is a placeholder page and add a link to the GH issue that's tracking the task of writing the actual content.